tlair/src/test/scala/at/forsyte/apalache/tla/lir/io/TestCounterexampleWriter.scala

```diff
@@ -83,37 +86,38 @@ class TestCounterexampleWriter extends FunSuite {
           |\* Created by Apalache on DATETIME
           |\* https://github.com/informalsystems/apalache
           |""".stripMargin
-      )
-    }
+    )
+  }
 
   test("two steps with conjunction") {
-    compare("tla",
-      new TlaModule("test",List()),
-      and(gt(name("x"), int(1)), eql(name("y"), int(10))),
-      List(
-        ("",Map("x" -> int(0), "y" -> int(8))),
-        ("Trans1", Map("x" -> int(1), "y" -> int(9))),
-        ("Trans2", Map("x" -> int(2), "y" -> int(10)))
-      ),
-      """------------------------- MODULE counterexample -------------------------
+    compare(
+        "tla",
+        new TlaModule("test", List()),
+        and(gt(name("x"), int(1)), eql(name("y"), int(10))),
+        List(
+            ("", Map("x" -> int(0), "y" -> int(8))),
+            ("Trans1", Map("x" -> int(1), "y" -> int(9))),
+            ("Trans2", Map("x" -> int(2), "y" -> int(10)))
+        ),
+        """------------------------- MODULE counterexample -------------------------
         |
         |EXTENDS test
         |
-        |(* Initial state *)
+        |(* Constant initialization state *)
         |
-        |State1 ==
+        |StateConst ==
```

Comment:
again, the constant initializer initializes a variable here, instead of a constant